linux/scripts/upload-to-debian.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+# Upload the latest release to mentors.debian.net. Call from stable-* branch.
+# Usage: upload-to-debian.sh -k <Debian signing key>
+# Requires an entry in ~/.dput.cf:
+# ```
+# [mentors]
+# fqdn = mentors.debian.net
+# incoming = /upload
+# method = https
+# allow_unsigned_uploads = 0
+# progress_indicator = 2
+# allowed_distributions = .*
+# ```
+
+set -e
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+PROGRAM_NAME="$(basename "$0")"
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m' # No Color
+
+error()
+{
+	echo -e "${RED}$PROGRAM_NAME: $1${NC}" >&2
+}
+
+log()
+{
+	echo -e "${GREEN}$PROGRAM_NAME: $1${NC}"
+}
```

Comment:
I think, source shellHelperFunctions.sh and use its defined colors:

https://github.com/keymanapp/keyman/blob/be7c9c58a9e81f596a64065cbbb0a58b786d9446/resources/shellHelperFunctions.sh#L85-L114

But we can probably improve what we have in shellHelperFunctions.sh!

Rename `PROGRAM_NAME` to `THIS_SCRIPT_NAME`, put that definition into build-utils.sh (this matches `THIS_SCRIPT` and `THIS_SCRIPT_PATH`.

Then, some of this may belong in our shellHelperFunctions.sh include, although probably with less generic names:

* rename `log` to `log_message` (or you may choose to use `echo_heading`?)
* rename `error` to `log_error`
* add `log_warning` (don't change `warn()` because it is used elsewhere)

```bash
log_message() {
  echo "${COLOR_GREEN}$THIS_SCRIPT_NAME: $*${COLOR_RESET}"
}

log_error() {
  echo "${COLOR_RED}$THIS_SCRIPT_NAME: $*${COLOR_RESET}"
}

log_warning() {
  echo "${COLOR_YELLOW}$THIS_SCRIPT_NAME: $*${COLOR_RESET}"
}
```

Note: we already have a chore to unify build-utils.sh and shellHelperFunctions.sh -- don't worry about doing that for now.